pkg/kubectl/cmd/set/set_env.go

```diff
@@ -149,7 +149,7 @@ func NewCmdEnv(f cmdutil.Factory, in io.Reader, out, errout io.Writer) *cobra.Co
 	cmd.Flags().StringP("from", "", "", "The name of a resource from which to inject environment variables")
 	cmd.Flags().StringP("prefix", "", "", "Prefix to append to variable names")
 	cmd.Flags().StringArrayVarP(&options.EnvParams, "env", "e", options.EnvParams, "Specify a key-value pair for an environment variable to set into each container.")
-	cmd.Flags().BoolVar(&options.List, "list", options.List, "If true, display the environment and any changes in the standard format")
+	cmd.Flags().BoolVar(&options.List, "list", options.List, "If true, display the environment and any changes in the standard format. this flag will removed when we have kubectl view image.")
```

Comment:
should be "view env"

Reply:
i will fix this in PR #52243, thanks

Reply:
Please, open a separate one since `kubectl set probe` won't make it for 1.8 but we could sneak in the typo fix.

Reply:
open pr #52248